src/librustc_mir/borrow_check/nll/region_infer/mod.rs

```diff
@@ -531,13 +598,188 @@ impl<'tcx> RegionInferenceContext<'tcx> {
             }
         }
 
+        // Now take member constraints into account
+        let member_constraints = self.member_constraints.clone();
+        for m_c_i in member_constraints.indices(scc_a) {
+            self.apply_member_constraint(
+                scc_a,
+                m_c_i,
+                member_constraints.choice_regions(m_c_i),
+            );
+        }
+
         debug!(
             "propagate_constraint_sccs: scc_a = {:?} has value {:?}",
             scc_a,
             self.scc_values.region_value_str(scc_a),
         );
     }
 
+    /// Invoked for each `member R0 of [R1..Rn]` constraint.
```

Comment:
```suggestion
    /// Invoked for each `R0 member of [R1..Rn]` constraint.
```
(Might want to look over how you've used this elsewhere)